docs/ref/models/fields.txt

```diff
@@ -1150,6 +1150,44 @@ when :attr:`~django.forms.Field.localize` is ``False`` or
     information on the difference between the two, see Python's documentation
     for the :mod:`decimal` module.
 
+``GeneratedField``
+------------------
+
+.. class:: GeneratedField(expression, output_field=None, db_persist=True, **kwargs)
+
+A field which is computed based on other fields in the model. Uses the database's
+``GENERATED ALWAYS AS`` syntax. Similar to a database view, this field is managed
+and updated by the database itself.
```

Comment:
Slight reordering of details, since SQL is a bit more of an implementation detail.

```suggestion
A field which is computed based on other fields in the model. Similar to a
database view, this field is managed and updated by the database itself. Uses
the ``GENERATED`` SQL syntax.
```

Also could we add an example after this paragraph? Just a basic use case, maybe from the tests, would really help users see what this does.